### DISCUSSION TITLE
안녕하세요!

### DISCUSSION THREAD

**Discussion by L4N14KE4** [learningssik/pull-request#117] [Q&A]
블로그 보고 왔습니다!

**Comment by learningssik** [comment]
안녕하세요 감사합니다!

**Reply by L4N14KE4** [reply]
현재 계정으로 galaxy brain을 얻고자 하면, 제가 만든 Q&A가 아니라, 선생님께서 만든 Q&A에 답 드려야 하는 걸까요?

**Reply by learningssik** [reply]
네 제가 만든 Q&A에서 채택을 받으셔야 합니다. 원하시면 제가 만들어드리겠습니다!

**Reply by L4N14KE4** [reply]
Q&A 만들어주시면 감사하겠습니다!
